Core/include/Acts/Seeding/GbtsDataStorage.hpp

```diff
@@ -29,20 +29,25 @@ struct GbtsSP {
   const space_point_t *SP;  // want inside to have pointer
   int gbtsID;
   int combined_ID;
-  GbtsSP(const space_point_t *sp, int id, int combined_id)
-      : SP(sp), gbtsID(id), combined_ID{combined_id} {
-    if (SP->sourceLinks().size() == 1) {  // pixels have 1 SL
-      m_isPixel = true;
-    } else {
-      m_isPixel = false;
-    }
-    m_phi = std::atan(SP->x() / SP->y());
+  bool m_isPixel;
+  float m_phi;
+  float m_r;
+  float m_ClusterWidth;
+  GbtsSP(const space_point_t *sp, int id, int combined_id, bool isPixel,
+         float ClusterWidth)
+      : SP(sp),
+        gbtsID(id),
+        combined_ID{combined_id},
+        m_isPixel(isPixel),
+        m_ClusterWidth(ClusterWidth) {
+    m_phi = std::atan2(SP->y(), SP->x());
+    m_r = std::sqrt((SP->x() * SP->x()) + (SP->y() * SP->y()));
   };
   bool isPixel() const { return m_isPixel; }
   bool isSCT() const { return !m_isPixel; }
```

Comment:
```suggestion
  bool isStrip() const { return !m_isPixel; }
```
Also need to change [SeedFinderGbts.ipp#L680](https://github.com/acts-project/acts/blob/eae98cc5c25993b1a82581f22d896185341a4dbd/Core/include/Acts/Seeding/SeedFinderGbts.ipp#L680).

This is only used in LRT mode, so probably doesn't work yet. But it might be best to have a less anachronistic ATLAS-specific name.